securedrop/source_templates/generate.html

```diff
@@ -39,12 +39,12 @@
 <div class="pull-right">
 <form id="create-form" method="post" action="/create" autocomplete="off">
   <input name="csrf_token" type="hidden" value="{{ csrf_token() }}">
-  <a id="already-have-codename" href="{{ url_for('main.login') }}" class="sd-button btn secondary">{{ gettext('ALREADY HAVE A CODENAME?') }}</a>
   <button type="submit" class="sd-button btn primary" id="continue-button">
     <img class="icon off-hover" src="{{ url_for('static', filename='i/font-awesome/fa-arrow-circle-o-right-white.png') }}" width="17px" height="17px">
     <img class="icon on-hover" src="{{ url_for('static', filename='i/font-awesome/fa-arrow-circle-o-right-blue.png') }}" width="17px" height="17px">
-     {{ gettext('CONTINUE') }}
+     {{ gettext('USE NEW CODENAME') }}
```

Comment:
What about **GET NEW CODENAME** ? I don't feel strongly about that but **USE** suggest you have a codename already. What really happens is that you'll be given a codename. What do you think ?

Reply:
I was operating under the impression the codename earlier in the page becomes their assigned code name if they click this button.

Perhaps "Use this codename " would be even better, but "get" makes me think it'll be in the next page. Want to make it clear to the user that they should make note of the codename before proceeding.

Reply:
> I was operating under the impression the codename earlier in the page becomes ...

You are correct and I was confused. Sorry for the noise.